docs/rest_api.rst

```diff
@@ -471,6 +471,9 @@ Connection Management
    :reqjson int initial_channel_target: Number of channels to open proactively.
    :reqjson float joinable_funds_target: Fraction of funds that will be used to join channels opened by other participants.
 
+   ..note::
+   Currently, the API calls are blocking. This means that in the case of long running calls like ``join``, if other calls to ``join`` are made concurrently, they will block too and wait for the first call to finish. If a second call is made with different arguments, those arguments will be the final ones.
```

Comment:
The following statement:
`If a second call is made with different arguments, those arguments will be the final ones.`
is a little vague. Could it be expressed as
`If an API call is currently being processed by Raiden, all pending calls will be queued and processed with their passed API call argument. `